.gitignore

```diff
@@ -0,0 +1,58 @@
+# IDE files #
+#############
+.idea
+
+# Web application ignores #
+###########################
+/.web-server-pid
+/config/packages/parameters.yaml
+/config/packages/institutions.yaml
+/build/
+/var/*
+!/var/cache
+/var/cache/*
+!var/cache/.gitkeep
+/var/log/*
+!var/log/.gitkeep
+!/var/sessions
+/var/sessions/*
+!var/sessions/.gitkeep
+!var/SymfonyRequirements.php
+/vendor/
+/node_modules/
+/web/build
+.env
+
+# OS generated files #
```

Comment:
Just a comment of good practices with git. OS generated/IDE's files and directories like this and `.idea`  are better to be in a personal global ignore file. Because each developer has his preferences and this .gitignore file only should ignore things than belongs to the project.

Reply:
Yes, good point. I copied this from my companies set of defaults. This is more of a defence in depth approach. As not all contributors might have their preferences set correctly in their personal git settings.

Seeing we will probably the only contributors for now I'm removing them.